### PR TITLE
add new interoperability section

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

- Create `Interoperability` section 
- Move `XCM` content to `Interoperability` section
- Move `integrations/bridges` content to `Interoperability` section and rename to `Cross-Chain Protocols`
- Update image paths
- Update internal links so redirects aren't required, this prevents 302 status codes (for SEO purposes)

### Checklist

- [ ] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [ ] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

mkdocs: https://github.com/PureStake/moonbeam-mkdocs/pull/96
original PR: https://github.com/PureStake/moonbeam-docs/pull/499

### CODE REVIEW

File: builders/interoperability/index.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+title: 互操作性
+description: Learn about interoperability on Moonbeam by diving into how cross-consensus messaging (XCM) works and exploring available cross-chain protocols.
```
Comment (eshaben) [inline builders/interoperability/index.md:3]: @wuxinxin2000 can you please translate this description?

File: builders/interoperability/protocols/.pages [hunk 2]
```diff
@@ -1,4 +1,4 @@
-title: 桥
+title: Cross-Chain Protocols
```
Comment (eshaben) [inline builders/interoperability/protocols/.pages:1]: @wuxinxin2000 can you please translate this title?

File: builders/interoperability/protocols/index.md [hunk 3]
```diff
@@ -0,0 +1,7 @@
+---
+title: Cross-Chain Protocols
+description: Learn about the cross-chain protocols you can use to securely communicate and transfer assets between Moonbeam and any connected blockchain.
```
Comment (eshaben) [inline builders/interoperability/protocols/index.md:3]: @wuxinxin2000 can you please translate this title and description?

Comment (wuxinxin2000) [reply builders/interoperability/protocols/index.md:3]: “Learn about the cross-chain protocols you can use to securely communicate and transfer assets between Moonbeam and any connected blockchain.” could be translated into
"了解跨链协议，用以在Moonbeam和任意已建立连接的链之间安全地转移资产。"

File: builders/interoperability/protocols/.pages (hunk 2 above)
Comment (wuxinxin2000) [reply builders/interoperability/protocols/.pages:1]: ”Cross-Chain Protocols“ could be translated into "跨链协议".

File: builders/interoperability/index.md (hunk 1 above)
Comment (wuxinxin2000) [reply builders/interoperability/index.md:3]: "Learn about interoperability on Moonbeam by diving into how cross-consensus messaging (XCM) works and exploring available cross-chain protocols." could be translated into "通过深入了解跨链共识消息(XCM)的工作原理以及探索已有的跨链协议来学习Moonbeam上的互操作性。"

Comment (eshaben) [inline builders/interoperability/index.md:3]:
```suggestion
description: 通过深入了解跨链共识消息(XCM)的工作原理以及探索已有的跨链协议来学习Moonbeam上的互操作性。
```

File: builders/interoperability/protocols/.pages (hunk 2 above)
Comment (eshaben) [inline builders/interoperability/protocols/.pages:1]:
```suggestion
title: 跨链协议
```

File: builders/interoperability/protocols/index.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+---
+title: Cross-Chain Protocols
```
Comment (eshaben) [inline builders/interoperability/protocols/index.md:2]:
```suggestion
title: 跨链协议
```

File: builders/interoperability/protocols/index.md (hunk 3 above)
Comment (eshaben) [inline builders/interoperability/protocols/index.md:3]:
```suggestion
description: 了解跨链协议，用以在Moonbeam和任意已建立连接的链之间安全地转移资产
```

**Review by wuxinxin2000 — APPROVED** [review]

**Pull request merged**